docs/submitting.md

```diff
@@ -9,8 +9,6 @@ But please read these instructions carefully for a streamlined submission.
 - The software should be open source as per the [OSI definition](https://opensource.org/osd).
 - The software should have an **obvious** research application.
 - You should be a major contributor to the software you are submitting.
-- The software should be a significant contribution to the available open source software that either enables some new research challenges to be addressed or makes addressing research challenges significantly better (e.g., faster, easier, simpler).
-- The software should be feature-complete (no half-baked solutions) and designed for maintainable extension (not one-off modifications). **Minor ‘utility’ packages, including ‘thin’ API clients, and single-function packages are not acceptable**.
 - Your paper (`paper.md` and BibTeX files, plus any figures) must be hosted in a Git-based repository. Placing these items together with your software (rather than in a separate repository) is **strongly encouraged**.
 
```

Comment:
Suggestion: add "- Your paper must not discuss new research."

Reply:
@pdebuyl In what sense? The code could be doing something new, yes?

Reply:
As I understand it, in JOSS we do not review new methods and new discoveries ("results"). We review code that relies on established algorithms for instance. There was an editorial "out of scope" for such an article this week and I thought that it should thus be in our documentation.

Reply:
I'm not comfortable with saying "research" is disjoint from software development. Good software contains lots of innovation that some researchers parlay into publications while others don't feel it's worth their time to do so. IMO, it's entirely fair to describe innovative aspects of the software, including how it is faster, more accurate, easier to extend and to use, etc. If we feel it's necessary, we could say that JOSS does not evaluate novelty/correctness of domain results, though I haven't noticed such a thing being an issue and there is a cost to these criteria becoming too long/tedious.

Reply:
Perhaps we could say "The paper should not focus on new research results accomplished with the software."

Reply:
Instead of "new" maybe "unpublished (i.e. without peer-review)"?

Reply:
I like the wording of not evaluating the novelty/correctness of domain results, though it's true we can't say everything in the docs and still get people to read them. Maybe focusing on evaluating the code alone is the important part. Someone could publish in JOSS a new algorithm and subsequently publish on its scientific use in another domain-specific journal, in that order, so I think publishing something new is ok as long as we're clear what has been tested/evaluated as part of the JOSS process.

Reply:
OK, I've changed this to:

```
You paper should not focus on new research results accomplished with the software.
```